internal/errs/silencederror.go

```diff
@@ -0,0 +1,24 @@
+package errs
+
+import (
+	"errors"
+)
+
+type silencedError struct {
+	error
+}
+
+func Silence(err error) silencedError {
+	return silencedError{err}
```

Comment:
```suggestion
func Silence(err error) *silencedError {
	return &silencedError{err}
```

 Your methods are using pointers.